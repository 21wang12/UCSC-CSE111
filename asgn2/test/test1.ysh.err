yshell: cat: /dev/null: No such file or directory
