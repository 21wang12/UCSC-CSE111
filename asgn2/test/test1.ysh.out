./yshell build Apr 29 2022 19:04:34
% prompt my-prompt:
my-prompt: ls /
/:
     1       2  ./
     1       2  ../
my-prompt: ls
/:
     1       2  ./
     1       2  ../
my-prompt: cat /dev/null
my-prompt: exit 5
yshell: exit(5)
